openproblems/tasks/_cell_cell_communication/_common/metrics/__init__.py

```diff
@@ -1 +1,2 @@
 from .odds_ratio import odds_ratio
+from .prauc import prauc
```

Comment:
I believe the standard abbreviation here is `auprc` (area under the precision-recall curve)

Reply:
Agreed.